tests/integration/cli/action/test_action_classification.py

```diff
@@ -67,3 +68,12 @@ def test_otx_export(self, template, tmp_dir_path):
     def test_otx_eval_openvino(self, template, tmp_dir_path):
         tmp_dir_path = tmp_dir_path / "action_cls"
         otx_eval_openvino_testing(template, tmp_dir_path, otx_dir, args, threshold=1.0)
+
+    @e2e_pytest_component
+    @pytest.mark.skipif(TT_STABILITY_TESTS, reason="This is TT_STABILITY_TESTS")
```

Comment:
Because full dataset size is maximum batch size, it's hard to make an environment to make cuda oom.
I think current implementation makes sense because current one can check preparation step which is task specific and therefore easy to be broken as task code is modified.